catalog-rest-service/src/main/resources/json/schema/entity/services/connections/database/mssqlConnection.json

```diff
@@ -5,16 +5,33 @@
   "description": "Mssql Database Connection Config",
   "type": "object",
   "javaType": "org.openmetadata.catalog.services.connections.database.MssqlConnection",
-  "properties": {
-    "mssqlConnection": {
-      "description": "a boolean config value to enable/disable connection for a service.",
-      "type": "boolean",
-      "default": "true"
+  "definitions": {
+    "mssqlType": {
+      "description": "Service type.",
+      "type": "string",
+      "enum": [
+        "MSSQL"
+      ],
+      "default": "MSSQL"
     },
-    "sqlDriverScheme": {
+    "mssqlScheme": {
       "description": "SQLAlchemy driver scheme options.",
       "type": "string",
-      "enum": ["mssql+pyodbc", "mssql+pytds", "mssql+pymssql"],
+      "enum": [
+        "mssql+pymssql"
```

Comment:
@pmbrull idea here is to provide all options so that UI can display a select box with list of options and default is what it chooses but user can pick other options where required